.eslintrc.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
             "@typescript-eslint/ban-ts-comment": "off",
 
             "quotes": "off",
+            "indent": "off",
```

Comment:
I might be missing something :sweat_smile: Why turn off the indent rule? I find it quite helpful when my editor auto-fixes indentation...

Reply:
It's just a reflection of the fact that we have many files that don't follow it due to the way the indent JSX props... I'm open to better suggestions... 😄

Reply:
E.g. I could reformat the props to match indent rule, just wasn't sure if we wanted that or not...

Reply:
(The rule is already off for JS files above, I'm just copying it because the problem files are now in TSX.)

Reply:
Changing the indentation would be great imo, but I am not sure what others think :man_shrugging:

Reply:
Okay, I agree it's better fixed, I'll work on it. 🙂

Reply:
Okay, fixed now and rule restored.